core-compose/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+  ~ Copyright 2020 Square Inc.
```

Comment:
`2020`. It's like science fiction.

Reply:
We can write magically reactive code, but we can't figure out how to do basic medical testing at scale. It's exactly like science fiction 🤦‍♂️